2022-10-21 20:56:42.140 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-10-21 20:56:51.495 WARNING (MainThread) [homeassistant.components.switch] Platform broadlink not ready yet: None; Retrying in background in 30 seconds
2022-10-21 20:56:56.247 WARNING (MainThread) [homeassistant.config_entries] Config entry 'SpeedTest' for speedtestdotnet integration not ready yet: <urlopen error [Errno -3] Try again>; Retrying in background
2022-10-21 20:57:10.191 ERROR (MainThread) [homeassistant.components.homeassistant_alerts] Timeout fetching homeassistant_alerts data
2022-10-21 20:57:15.011 ERROR (MainThread) [custom_components.hacs] <QueueManager> <Plugin ljmerza/reddit-card> Repository structure for 1.0.1 is not compliant
2022-10-21 20:57:56.257 WARNING (MainThread) [homeassistant.components.automation.dan_ji_yi_lou_wo_shi_deng] 一楼卧室灯单击: Already running
2022-10-21 20:57:58.706 WARNING (MainThread) [homeassistant.components.automation.dan_ji_yi_lou_wo_shi_deng] 一楼卧室灯单击: Already running
2022-10-21 20:58:00.121 WARNING (MainThread) [homeassistant.components.automation.dan_ji_yi_lou_wo_shi_deng] 一楼卧室灯单击: Already running
2022-10-21 20:58:01.584 WARNING (MainThread) [homeassistant.components.automation.chang_an_yi_lou_wo_shi_deng] 一楼卧室灯长按: Already running
2022-10-21 20:58:03.204 WARNING (MainThread) [homeassistant.components.automation.dan_ji_yi_lou_wo_shi_deng] 一楼卧室灯单击: Already running
2022-10-21 20:58:52.542 WARNING (MainThread) [homeassistant.components.automation.dan_ji_yi_lou_wo_shi_deng] 一楼卧室灯单击: Already running
2022-10-21 21:09:40.579 WARNING (MainThread) [homeassistant.components.automation.chang_an_yi_lou_wo_shi_deng] 一楼卧室灯长按: Already running
2022-10-21 21:11:58.769 WARNING (MainThread) [homeassistant.components.synology_dsm.common] Connection error during update, fallback by reloading the entry
2022-10-21 21:12:04.849 WARNING (MainThread) [homeassistant.config_entries] Config entry '192.168.1.8' for synology_dsm integration not ready yet: ConnectionError = <urllib3.connection.HTTPConnection object at 0x7f6e2d6aa0>: Failed to establish a new connection: [Errno 113] Host is unreachable; Retrying in background
2022-10-21 21:58:36.436 WARNING (MainThread) [homeassistant.helpers.entity] Update of switch.plug_158d00015d1bf5 is taking over 10 seconds
2022-10-21 21:58:36.451 ERROR (SyncWorker_3) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-21 21:58:36.465 ERROR (SyncWorker_3) [xiaomi_gateway] No data in response from hub None
2022-10-21 22:57:36.593 WARNING (MainThread) [homeassistant.helpers.entity] Update of switch.plug_158d00015d1bf5 is taking over 10 seconds
2022-10-21 22:57:36.601 ERROR (SyncWorker_8) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-21 22:57:36.604 ERROR (SyncWorker_8) [xiaomi_gateway] No data in response from hub None
2022-10-21 23:34:36.703 WARNING (MainThread) [homeassistant.helpers.entity] Update of switch.plug_158d00015d1bf5 is taking over 10 seconds
2022-10-21 23:34:36.712 ERROR (SyncWorker_4) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-21 23:34:36.719 ERROR (SyncWorker_4) [xiaomi_gateway] No data in response from hub None
2022-10-21 23:43:06.727 WARNING (MainThread) [homeassistant.helpers.entity] Update of binary_sensor.door_window_sensor_158d000128fa46 is taking over 10 seconds
2022-10-21 23:43:06.734 ERROR (SyncWorker_7) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-21 23:43:06.743 ERROR (SyncWorker_7) [xiaomi_gateway] No data in response from hub None
2022-10-22 00:03:57.018 ERROR (MainThread) [homeassistant.components.broadlink.updater] Error fetching s2 (MP1-1K4S at 192.168.1.15) data: [Errno -4000] Network timeout: No response received within 5s
2022-10-22 00:07:01.701 ERROR (MainThread) [custom_components.hacs] <QueueManager> <Plugin ljmerza/reddit-card> Repository structure for 1.0.1 is not compliant
2022-10-22 00:42:06.909 WARNING (MainThread) [homeassistant.helpers.entity] Update of binary_sensor.door_window_sensor_158d000128fa46 is taking over 10 seconds
2022-10-22 00:42:06.918 ERROR (SyncWorker_3) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 00:42:06.923 ERROR (SyncWorker_3) [xiaomi_gateway] No data in response from hub None
2022-10-22 01:16:37.009 WARNING (MainThread) [homeassistant.helpers.entity] Update of binary_sensor.door_window_sensor_158d000128fa46 is taking over 10 seconds
2022-10-22 01:16:37.018 ERROR (SyncWorker_7) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 01:16:37.021 ERROR (SyncWorker_7) [xiaomi_gateway] No data in response from hub None
2022-10-22 01:43:07.080 WARNING (MainThread) [homeassistant.helpers.entity] Update of binary_sensor.door_window_sensor_158d000128fa46 is taking over 10 seconds
2022-10-22 01:43:07.090 ERROR (SyncWorker_0) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 01:43:07.094 ERROR (SyncWorker_0) [xiaomi_gateway] No data in response from hub None
2022-10-22 01:47:37.092 WARNING (MainThread) [homeassistant.helpers.entity] Update of switch.plug_158d00015d1bf5 is taking over 10 seconds
2022-10-22 01:47:37.100 WARNING (MainThread) [homeassistant.helpers.entity] Update of binary_sensor.door_window_sensor_158d000128fa46 is taking over 10 seconds
2022-10-22 01:47:37.104 ERROR (SyncWorker_6) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 01:47:37.105 ERROR (SyncWorker_0) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 01:47:37.115 ERROR (SyncWorker_6) [xiaomi_gateway] No data in response from hub None
2022-10-22 01:47:37.122 ERROR (SyncWorker_0) [xiaomi_gateway] No data in response from hub None
2022-10-22 02:49:07.321 WARNING (MainThread) [homeassistant.helpers.entity] Update of switch.plug_158d00015d1bf5 is taking over 10 seconds
2022-10-22 02:49:07.329 ERROR (SyncWorker_7) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 02:49:07.335 ERROR (SyncWorker_7) [xiaomi_gateway] No data in response from hub None
2022-10-22 02:57:06.648 ERROR (MainThread) [custom_components.hacs] <QueueManager> <Plugin ljmerza/reddit-card> Repository structure for 1.0.1 is not compliant
2022-10-22 03:05:59.011 ERROR (MainThread) [homeassistant.components.broadlink.updater] Error fetching s3 (MP1-1K4S at 192.168.1.16) data: [Errno -4000] Network timeout: No response received within 5s
2022-10-22 03:22:04.006 ERROR (MainThread) [homeassistant.components.broadlink.updater] Error fetching s3 (MP1-1K4S at 192.168.1.16) data: [Errno -4000] Network timeout: No response received within 5s
2022-10-22 03:56:37.514 WARNING (MainThread) [homeassistant.helpers.entity] Update of switch.plug_158d00015d1bf5 is taking over 10 seconds
2022-10-22 03:56:37.522 WARNING (MainThread) [homeassistant.helpers.entity] Update of binary_sensor.door_window_sensor_158d000128fa46 is taking over 10 seconds
2022-10-22 03:56:37.524 ERROR (SyncWorker_8) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 03:56:37.524 ERROR (SyncWorker_4) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 03:56:37.531 ERROR (SyncWorker_8) [xiaomi_gateway] No data in response from hub None
2022-10-22 03:56:37.537 ERROR (SyncWorker_4) [xiaomi_gateway] No data in response from hub None
2022-10-22 04:00:00.025 WARNING (MainThread) [homeassistant.helpers.service] Unable to find referenced devices cdfee5b5c3de20f51b5c596e260e8af2 or it is/they are currently not available
2022-10-22 04:02:02.008 ERROR (MainThread) [homeassistant.components.broadlink.updater] Error fetching s2 (MP1-1K4S at 192.168.1.15) data: [Errno -4000] Network timeout: No response received within 5s
2022-10-22 04:06:37.544 WARNING (MainThread) [homeassistant.helpers.entity] Update of switch.plug_158d00015d1bf5 is taking over 10 seconds
2022-10-22 04:06:37.552 ERROR (SyncWorker_0) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 04:06:37.555 ERROR (SyncWorker_0) [xiaomi_gateway] No data in response from hub None
2022-10-22 04:50:07.675 WARNING (MainThread) [homeassistant.helpers.entity] Update of switch.plug_158d00015d1bf5 is taking over 10 seconds
2022-10-22 04:50:07.683 WARNING (MainThread) [homeassistant.helpers.entity] Update of binary_sensor.door_window_sensor_158d000128fa46 is taking over 10 seconds
2022-10-22 04:50:07.686 ERROR (SyncWorker_6) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 04:50:07.687 ERROR (SyncWorker_7) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 04:50:07.697 ERROR (SyncWorker_6) [xiaomi_gateway] No data in response from hub None
2022-10-22 04:50:07.700 ERROR (SyncWorker_7) [xiaomi_gateway] No data in response from hub None
2022-10-22 05:07:10.006 ERROR (MainThread) [homeassistant.components.broadlink.updater] Error fetching s3 (MP1-1K4S at 192.168.1.16) data: [Errno -4000] Network timeout: No response received within 5s
2022-10-22 05:44:07.841 WARNING (MainThread) [homeassistant.helpers.entity] Update of switch.plug_158d00015d1bf5 is taking over 10 seconds
2022-10-22 05:44:07.849 ERROR (SyncWorker_4) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 05:44:07.855 ERROR (SyncWorker_4) [xiaomi_gateway] No data in response from hub None
2022-10-22 05:45:37.844 WARNING (MainThread) [homeassistant.helpers.entity] Update of binary_sensor.door_window_sensor_158d000128fa46 is taking over 10 seconds
2022-10-22 05:45:37.852 ERROR (SyncWorker_1) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 05:45:37.858 ERROR (SyncWorker_1) [xiaomi_gateway] No data in response from hub None
2022-10-22 05:46:37.849 WARNING (MainThread) [homeassistant.helpers.entity] Update of switch.plug_158d00015d1bf5 is taking over 10 seconds
2022-10-22 05:46:37.857 WARNING (MainThread) [homeassistant.helpers.entity] Update of binary_sensor.door_window_sensor_158d000128fa46 is taking over 10 seconds
2022-10-22 05:46:37.859 ERROR (SyncWorker_3) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 05:46:37.860 ERROR (SyncWorker_0) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 05:46:37.868 ERROR (SyncWorker_3) [xiaomi_gateway] No data in response from hub None
2022-10-22 05:46:37.874 ERROR (SyncWorker_0) [xiaomi_gateway] No data in response from hub None
2022-10-22 05:47:07.848 WARNING (MainThread) [homeassistant.helpers.entity] Update of switch.plug_158d00015d1bf5 is taking over 10 seconds
2022-10-22 05:47:07.855 WARNING (MainThread) [homeassistant.helpers.entity] Update of binary_sensor.door_window_sensor_158d000128fa46 is taking over 10 seconds
2022-10-22 05:47:07.856 ERROR (SyncWorker_7) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 05:47:07.856 ERROR (SyncWorker_4) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 05:47:07.864 ERROR (SyncWorker_7) [xiaomi_gateway] No data in response from hub None
2022-10-22 05:47:07.870 ERROR (SyncWorker_4) [xiaomi_gateway] No data in response from hub None
2022-10-22 05:57:01.357 ERROR (SyncWorker_3) [homeassistant.components.xiaomi.device_tracker] Connection failed with http code 502
NoneType: None
2022-10-22 05:57:03.220 ERROR (SyncWorker_3) [homeassistant.components.xiaomi.device_tracker] Invalid response: [<Response [502]>] at url: [http://192.168.1.1/cgi-bin/luci/api/xqsystem/login] with data [{'username': 'admin', 'password': 'miaoyanginfo'}]
2022-10-22 06:07:01.667 ERROR (MainThread) [custom_components.hacs] <QueueManager> <Plugin ljmerza/reddit-card> Repository structure for 1.0.1 is not compliant
2022-10-22 06:09:07.910 WARNING (MainThread) [homeassistant.helpers.entity] Update of binary_sensor.door_window_sensor_158d000128fa46 is taking over 10 seconds
2022-10-22 06:09:07.918 ERROR (SyncWorker_8) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 06:09:07.921 ERROR (SyncWorker_8) [xiaomi_gateway] No data in response from hub None
2022-10-22 06:21:15.009 ERROR (MainThread) [homeassistant.components.broadlink.updater] Error fetching s3 (MP1-1K4S at 192.168.1.16) data: [Errno -4000] Network timeout: No response received within 5s
2022-10-22 06:53:27.501 WARNING (MainThread) [homeassistant.helpers.service] Unable to find referenced entities switch.er_lou_wo_shi_deng or it is/they are currently not available
2022-10-22 07:06:21.008 ERROR (MainThread) [homeassistant.components.broadlink.updater] Error fetching s3 (MP1-1K4S at 192.168.1.16) data: [Errno -4000] Network timeout: No response received within 5s
2022-10-22 07:48:32.986 WARNING (MainThread) [homeassistant.helpers.service] Unable to find referenced entities switch.er_lou_wo_shi_deng or it is/they are currently not available
2022-10-22 07:57:04.801 WARNING (MainThread) [homeassistant.components.websocket_api.http.connection] [546396636160] Disconnected: Did not receive auth message within 10 seconds
2022-10-22 08:30:22.493 WARNING (MainThread) [homeassistant.components.automation.2f_keting_switch_on_single_press_2] 一楼客厅单击: Already running
2022-10-22 08:31:10.642 WARNING (MainThread) [homeassistant.components.automation.2f_keting_switch_on_long_press_2] 一楼客厅长按: Already running
2022-10-22 08:57:03.653 ERROR (MainThread) [custom_components.hacs] <QueueManager> <Plugin ljmerza/reddit-card> Repository structure for 1.0.1 is not compliant
2022-10-22 09:00:00.040 WARNING (MainThread) [homeassistant.helpers.service] Unable to find referenced entities switch.er_lou_wo_shi_deng, switch.yi_lou_wo_shi_deng or it is/they are currently not available
2022-10-22 09:05:38.468 WARNING (MainThread) [homeassistant.helpers.entity] Update of switch.plug_158d00015d1bf5 is taking over 10 seconds
2022-10-22 09:05:38.474 WARNING (MainThread) [homeassistant.helpers.entity] Update of binary_sensor.door_window_sensor_158d000128fa46 is taking over 10 seconds
2022-10-22 09:05:38.477 ERROR (SyncWorker_7) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 09:05:38.478 ERROR (SyncWorker_5) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 09:05:38.484 ERROR (SyncWorker_7) [xiaomi_gateway] No data in response from hub None
2022-10-22 09:05:38.491 ERROR (SyncWorker_5) [xiaomi_gateway] No data in response from hub None
2022-10-22 09:23:57.022 ERROR (MainThread) [homeassistant.components.broadlink.updater] Error fetching s1 (MP1-1K4S at 192.168.1.14) data: [Errno -4000] Network timeout: No response received within 5s
2022-10-22 09:55:08.639 WARNING (MainThread) [homeassistant.helpers.entity] Update of binary_sensor.door_window_sensor_158d000128fa46 is taking over 10 seconds
2022-10-22 09:55:08.648 ERROR (SyncWorker_1) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 09:55:08.654 ERROR (SyncWorker_1) [xiaomi_gateway] No data in response from hub None
2022-10-22 09:58:07.009 ERROR (MainThread) [homeassistant.components.broadlink.updater] Error fetching s2 (MP1-1K4S at 192.168.1.15) data: [Errno -4000] Network timeout: No response received within 5s
2022-10-22 10:06:02.008 ERROR (MainThread) [homeassistant.components.broadlink.updater] Error fetching s1 (MP1-1K4S at 192.168.1.14) data: [Errno -4000] Network timeout: No response received within 5s
2022-10-22 10:20:31.074 ERROR (MainThread) [aiohttp.server] Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/aiohttp/web_protocol.py", line 334, in data_received
    messages, upgraded, tail = self._request_parser.feed_data(data)
  File "aiohttp/_http_parser.pyx", line 551, in aiohttp._http_parser.HttpParser.feed_data
aiohttp.http_exceptions.BadStatusLine: 400, message="Bad status line 'Invalid method encountered'"
2022-10-22 10:30:12.009 ERROR (MainThread) [homeassistant.components.broadlink.updater] Error fetching s2 (MP1-1K4S at 192.168.1.15) data: [Errno -4000] Network timeout: No response received within 5s
2022-10-22 12:07:01.779 ERROR (MainThread) [custom_components.hacs] <QueueManager> <Plugin ljmerza/reddit-card> Repository structure for 1.0.1 is not compliant
2022-10-22 12:20:09.057 WARNING (MainThread) [homeassistant.helpers.entity] Update of switch.plug_158d00015d1bf5 is taking over 10 seconds
2022-10-22 12:20:09.066 ERROR (SyncWorker_8) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 12:20:09.073 ERROR (SyncWorker_8) [xiaomi_gateway] No data in response from hub None
2022-10-22 12:51:27.017 ERROR (MainThread) [homeassistant.components.broadlink.updater] Error fetching s3 (MP1-1K4S at 192.168.1.16) data: [Errno -4000] Network timeout: No response received within 5s
2022-10-22 13:03:39.185 WARNING (MainThread) [homeassistant.helpers.entity] Update of binary_sensor.door_window_sensor_158d000128fa46 is taking over 10 seconds
2022-10-22 13:03:39.192 ERROR (SyncWorker_7) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 13:03:39.195 ERROR (SyncWorker_7) [xiaomi_gateway] No data in response from hub None
2022-10-22 13:16:32.009 ERROR (MainThread) [homeassistant.components.broadlink.updater] Error fetching s3 (MP1-1K4S at 192.168.1.16) data: [Errno -4000] Network timeout: No response received within 5s
2022-10-22 13:20:39.228 WARNING (MainThread) [homeassistant.helpers.entity] Update of binary_sensor.door_window_sensor_158d000128fa46 is taking over 10 seconds
2022-10-22 13:20:39.237 ERROR (SyncWorker_6) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 13:20:39.243 ERROR (SyncWorker_6) [xiaomi_gateway] No data in response from hub None
2022-10-22 13:27:09.244 WARNING (MainThread) [homeassistant.helpers.entity] Update of binary_sensor.door_window_sensor_158d000128fa46 is taking over 10 seconds
2022-10-22 13:27:09.252 ERROR (SyncWorker_5) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 13:27:09.259 ERROR (SyncWorker_5) [xiaomi_gateway] No data in response from hub None
2022-10-22 13:38:46.726 WARNING (MainThread) [homeassistant.components.automation.2f_keting_switch_on_double_press_2] 一楼客厅双击: Already running
2022-10-22 13:40:28.070 ERROR (SyncWorker_8) [homeassistant.components.xiaomi.device_tracker] Connection failed with http code 502
NoneType: None
2022-10-22 13:40:35.208 ERROR (SyncWorker_8) [homeassistant.components.xiaomi.device_tracker] Connection failed with http code 502
NoneType: None
2022-10-22 13:56:09.312 WARNING (MainThread) [homeassistant.helpers.entity] Update of binary_sensor.door_window_sensor_158d000128fa46 is taking over 10 seconds
2022-10-22 13:56:09.319 ERROR (SyncWorker_2) [xiaomi_gateway] Cannot connect to gateway 286c07f0c4e5
2022-10-22 13:56:09.322 ERROR (SyncWorker_2) [xiaomi_gateway] No data in response from hub None
2022-10-22 13:58:08.005 ERROR (MainThread) [homeassistant.components.broadlink.updater] Error fetching s1 (MP1-1K4S at 192.168.1.14) data: [Errno -4000] Network timeout: No response received within 5s
